src/main/java/com/williamfiset/algorithms/other/RangeUpdate.java

```diff
@@ -9,10 +10,19 @@
 
 public class RangeUpdate {
 
+  private int[] array;
+  private int[] differenceArray;
```

Comment:
I have internalized the arrays so the user never has to pass them as arguments to update